MoEngagePluginBase.podspec

```diff
@@ -22,6 +22,6 @@ Pod::Spec.new do |s|
     s.source_files = 'MoEngagePluginBase/**/*'
     s.swift_version = '5.0'
     s.frameworks = 'UIKit', 'Foundation', 'UserNotifications'
-    s.dependency 'MoEngage-iOS-SDK', '>= 8.4.0', '< 8.5.0'
-    s.dependency 'MoEngageInApp', '>= 3.4.0', '< 3.5.0'
+    s.dependency 'MoEngage-iOS-SDK', '>= 9.1.0', '< 9.2.0'
```

Comment:
@RakshithaAcharya As we have release 9.2.x should we updating it to the latest version?

Reply:
@deepa-iOS-SDK  ill update in main branch.. when I had raised 9.2.0 was not yet released